src/hooks/use-aggrid-sort.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useCallback, useState } from 'react';
+
+export const SORT_WAYS = {
+    asc: 1,
+    desc: -1,
+};
+
+export const useAgGridSort = (dataKeyToSortKey, initSortConfig) => {
+    const { colKey, sortWay } = initSortConfig || {};
+
+    const [sortSelector, setSortSelector] = useState(
+        colKey && sortWay
+            ? { sortKeysWithWeightAndDirection: { [colKey]: sortWay } }
+            : {}
```

Comment:
create method returning this object, same for lines 37-41

Reply:
done, thank you